### ISSUE TITLE
reporting unresolved symbol for test symbols in cljc files

### ISSUE THREAD

**Issue by lread** [borkdude/clj-kondo#335]
**version**
clj-kondo v2019.07.06-alpha-SNAPSHOT (on 2019-07-09)

**platform**
macOS

**problem**
clj-kondo will inform of missing symbols for `thrown-with-msg?` and missing symbols related to `are` in cljc files

**repro**
`borkdude_test.cljc`
```clojure
(ns borkdude-test
  (:require [clojure.test :refer [deftest is are]])
  #?(:clj (:import clojure.lang.ExceptionInfo)))

(deftest are-test
  (are [?a ?b]
      (is (= ?a (dec ?b)))
    1 2
    10 11
    14 15))

(deftest thown-with-msg-test
  (is (thrown-with-msg? ExceptionInfo #"uh oh" (throw (ex-info "uh oh" {})))))
```

**expected behavior**
clj-kondo reports no problems

**actual behavior**
clj-kondo reports missing symbols for `?a`, `?b` and `thrown-with-msg?`

**work-around**
add the following to your `clj-kondo/config.edn`:
```clojure
{:linters
 {:unresolved-symbol
  {:exclude [(cljs.test/are)
             (cljs.test/is [thrown-with-msg?])]}}}
```

**Comment by lread** [comment]
Very much related: `thrown` should be added to exclude vector for both `clojure.test/is` and `cljs.test/is`

**Comment by borkdude** [comment]
@lread Can be tested with: https://4435-176829714-gh.circle-artifacts.com/0/release/clj-kondo-2019.07.06-alpha-SNAPSHOT-macos-amd64.zip

**Comment by borkdude** [comment]
Fixed with https://github.com/borkdude/clj-kondo/commit/b82e5b89d47ea4f1c85d92c29b5323c11e94f107

**Comment by borkdude** [comment]
Fixed with https://github.com/borkdude/clj-kondo/commit/c795c4471b9e1ae495ec0bdc9faf9009c679eec0

**Comment by lread** [comment]
thanks, I tried it and it works!
